model_compression_toolkit/target_platform_capabilities/constants.py

```diff
@@ -21,10 +21,10 @@
 
 
 # Supported TP models names:
-DEFAULT_TP_MODEL = 'default'
```

Comment:
I’m debating this change... this is a breaking change for any user that tried to use the generate_tpc API with IMX500_TP_MODEL
Not sure that we want to break every potential user with this, and we can wait with this to the extraction of the IMX500 TPCs to the external repo.